packages/api/src/promo/queries/get_promos.sql

```diff
@@ -0,0 +1,12 @@
+SELECT
+  promoid AS "id",
+  code,
+  sizeinmb AS "storageInMB",
+  expiresdt AS "expirationTimestamp",
+  remaininguses AS "remainingUses",
+  status,
+  type,
+  createddt AS "createdAt",
+  updateddt AS "updatedAt"
```

Comment:
I wasn't certain whether `promoid`, `status`, and `type` should be returned here, as all are fairly internal (`status` and `type` are functionally unused). I decided to err on the side of exposing things since this is an admin endpoint.